setup.py

```diff
@@ -27,20 +27,10 @@
     python_requires=">=3.6.*",
     install_requires=[
         'clickhouse-driver~=0.1.2',
-        'apache-airflow~=1.10.6',
+        'apache-airflow>=2.0.0',
```

Comment:
```suggestion
        'apache-airflow~=2.0.1',
```

We may not be sure that 2.1.0 release won't break the plugin.